README.md

```diff
@@ -129,7 +123,7 @@ jobs:
 
     steps:
     - id: 'auth'
-      uses: 'google-github-actions/auth@v0'
+      uses: 'google-github-actions/auth@v1'
```

Comment:
Can you do a once over to make sure all references to v0 are updated to v1, including for this action. This is a chicken vs egg problem but we can update in one shot prior to releasing.

Reply:
sg, all updated to v1.